pkg/formatters/rawFormatter.go

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright contributors to the Galasa project
+ */
+package formatters
+
+import (
+	"strings"
+	"time"
+
+	"github.com/galasa.dev/cli/pkg/galasaapi"
+)
+
+// -----------------------------------------------------
+// Summary format.
+const (
+	RAW_FORMATTER_NAME = "raw"
+)
+
+type RawFormatter struct {
+}
+
+func NewRawFormatter() RunsFormatter {
+	return new(RawFormatter)
+}
+
+func (*RawFormatter) GetName() string {
+	return RAW_FORMATTER_NAME
+}
+
+func (*RawFormatter) IsNeedingMethodDetails() bool {
+	return false
+}
+
+func (*RawFormatter) FormatRuns(runs []galasaapi.Run, apiServerUrl string) (string, error) {
+	var result string = ""
+	var err error = nil
+	buff := strings.Builder{}
+
+	for _, run := range runs {
+
+		var duration string = ""
+
+		var startTimeStringForDuration time.Time
+		var endTimeStringForDuration time.Time
+
+		startTimeStringRaw := run.TestStructure.GetStartTime()
+		endTimeStringRaw := run.TestStructure.GetEndTime()
+
+		if len(startTimeStringRaw) > 0 {
```

Comment:
Is this code duplicated with the other formatters which work out time deltas ?

Reply:
Yes it is similar to code in the details formatter. I will refactor